src/http/persistent_subscriptions.rs

```diff
@@ -0,0 +1,170 @@
+use crate::grpc::Handle;
+use crate::{
+    ClientSettings, GetPersistentSubscriptionInfoOptions, ListPersistentSubscriptionsOptions,
+    PersistentSubscriptionInfo, ReplayParkedMessagesOptions,
+    RestartPersistentSubscriptionSubsystem,
+};
+
+/// Replays a persistent subscriptions parked events.
+pub(crate) async fn replay_parked_messages(
+    handle: &Handle,
+    http_client: &reqwest::Client,
+    settings: &ClientSettings,
+    stream_name: impl AsRef<str>,
+    group_name: impl AsRef<str>,
+    options: &ReplayParkedMessagesOptions,
+) -> crate::Result<()> {
+    let mut builder = http_client
+        .post(format!(
+            "{}/subscriptions/{}/{}/replayParked",
```

Comment:
Do you need to encode your stream and group name?  `my/great/stream` is a valid stream name.

Reply:
Done and added tests too